src/containers/ViewerPanel/index.tsx

```diff
@@ -261,13 +264,25 @@ class ViewerPanel extends React.Component<ViewerPanelProps, ViewerPanelState> {
         setIsLooping(!isLooping);
     }
 
+    public handleIncomingConvertedFile(data: TrajectoryFileInfo) {
+        const { receiveConvertedFile, conversionProcessingData } = this.props;
+        const fileId = conversionProcessingData.fileId + ".simularium";
+        const hasTitle =
+            data.trajectoryTitle !== undefined &&
+            data.trajectoryTitle.length > 0;
+        const title = hasTitle ? data.trajectoryTitle : fileId;
+        receiveConvertedFile({
+            name: fileId,
+            title: title,
+        });
+    }
+
     public onTrajectoryFileInfoChanged(data: TrajectoryFileInfo) {
-        const { conversionStatus, setConversionStatus } = this.props;
-        if (conversionStatus !== CONVERSION_INACTIVE) {
-            setConversionStatus({ status: CONVERSION_INACTIVE });
+        const { receiveTrajectory, simulariumController, conversionStatus } =
+            this.props;
+        if (conversionStatus === CONVERSION_ACTIVE) {
```

Comment:
I have a memory of these status's being an enum? Like the way we have for the `VIEWER_STATUS`?

Reply:
That code hasn't been put in PR yet for either viewer or conversion status, I have both changes in the same dev branch.